internal/commands/image/inspect.go

```diff
@@ -30,16 +30,17 @@ func muteDockerCli(dockerCli command.Cli) func() {
 func inspectCmd(dockerCli command.Cli) *cobra.Command {
 	var opts inspectOptions
 	cmd := &cobra.Command{
-		Use:   "inspect [APP_NAME] [OPTIONS]",
-		Short: "Shows metadata, parameters and a summary of the Compose file for a given application",
-		Example: `$ docker app inspect my-installed-app
-$docker app inspect my-app:1.0.0`,
+		Use:   "inspect [APP_IMAGE] [OPTIONS]",
+		Short: "Display detailed information on an App Image",
+		Example: `$ docker app image inspect myapp
+$ docker app image inspect myapp:1.0.0
+$ docker app image inspect myrepo/myapp:1.0.0`,
```

Comment:
Should we add an example with App Image id?

Reply:
Yes

Reply:
Added